lunes_cms/cms/models/sponsor.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .static import upload_sponsor_icons
+
+
+class Sponsor(models.Model):
+    """
+    Model to manage our sponsors that are shown in the app.
+    """
+
+    name = models.TextField(
```

Comment:
I think I find a TextField a bit to large. Maybe I'm not thinking of something here, but I think a CharField should be enough